lambdatimes/src/components/Content/Tab.js

```diff
@@ -18,5 +20,10 @@ const Tab = props => {
 };
 
 // Make sure you include PropTypes on your props.
+Tab.propTypes = {
+  tab: pt.string,
```

Comment:
Could mark these as `isRequired` as they are needed for the component to function properly. 👍